build-tools/packages/version-tools/src/index.ts

```diff
@@ -40,5 +40,5 @@ export {
 	VersionScheme,
 } from "./schemes";
 export { bumpRange, detectBumpType, isPrereleaseVersion, getPreviousVersions } from "./semver";
-export { getIsLatest, getSimpleVersion } from "./versions";
+export { getIsLatest, getSimpleVersion, isTestVersion } from "./versions";
```

Comment:
I think the two options I'm comfortable with are:

1. Move the isTestVersion function local to the command so it doesn't have to be exported and caveat it with a bunch of warnings that it's not a complete implementation. That way if it "breaks," the break can only affect the report-unrelease command, since the function is only there.
2. Add a bunch of tests to version-tools to make sure that the isTestVersion function handles all the version kinds that we produce. Those test cases will force some decisions. For example, why is 2.1.0-123456-test not a "test version?"

I don't know how time-critical this is, so while I prefer option 2 I am ok with option 1 since what can break is very limited in that case.

Reply:
I agree with Tyler here.